### PR TITLE
fix hotplug legacy pci device into wrong bus issue

### PR DESCRIPTION
**Pull request by xutian**
legacy pci device in Q35 type vm should be plug into pcie-pci-bridge and not pci.0

ID: 1808960, 1809396

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Test pass:
(1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.rtl8139.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.multi_nics_hotplug.nic_8139.q35: PASS (524.63 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.rtl8139.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.boot.q35: PASS (47.16 s)
Check the test log,rtl8139 nic is hotplug on pcie-pci-bridge,the result is expected.

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -1974,7 +1948,7 @@ def sort_key(dev):
 
                 # Handle the '-net nic' part
                 virtio = "virtio" in nic_model
-                parent_bus = _get_pci_bus(devices, nic_params, "nic", virtio)
+                parent_bus = self._get_pci_bus(nic_params, "nic", virtio)
```
Comment (yanan-fu) [inline virttest/qemu_vm.py]: @xutian  `e1000e` is a pcie device, but with `virtio = "virtio" in nic_model` is `False`,  which case `pcie=False`  in `_get_pci_bus`.  this device will be added to `pcie-pci-bridge-0`.

Comment (yanan-fu) [reply virttest/qemu_vm.py]: @xutian  It is an existing bug which is not related with this patch, we can fix with a separate commit:)

Comment (xutian) [reply virttest/qemu_vm.py]: fixed

Comment (yanan-fu) [reply virttest/qemu_vm.py]: @xutian Thanks, I am ok with the fix, but a new commit is more better, it is not related with hotplug, how do you think?

**Comment by yanan-fu** [comment]
Gating test pass, and test with boot vm with all the devices that be effected, check the device topology:
1. watchdog  i6300esb:  bus=pcie-pci-bridge-0
2. soundcard intel-hda:  bus=pcie-pci-bridge-0
3. virtio_rngs: rng-random:  pcie-root-port
4. usb qemu-xhci: pcie-root-port
5. disk: pcie-root-port
6. balloon: pcie-root-port
7. anaconda_log: pcie-root-port
8. cdrom: do not cover this
9. vio_9: do not cover this
For 8 and 9, from code review, the logic is same with before.

Acked-by: Yanan Fu <yfu@redhat.com>

**Comment by xutian** [comment]
thanks @yanan-fu and @yanglei-rh

**Pull request merged**